cloud-functions/assets/test_assets.py

```diff
@@ -0,0 +1,9 @@
+import json
+
+
+single_patient_bundle = json.load(open("../assets/single_patient_bundle.json", "r"))
+
+
+def test_single_patient_bundle():
+
+    assert single_patient_bundle is not None
```

Comment:
I'm curious what the purpose of this test is? Not necessarily opposed to it though. If we keep it though, I think the code on line 4 should moved within `test_single_patient_bundle()`.